regexes/client/mobile_apps.yml

```diff
@@ -54,17 +54,17 @@
   version: '$1'
 
 # Facebook Messenger (https://play.google.com/store/apps/details?id=com.facebook.orca)
-- regex: '(?:MessengerForiOS|MESSENGER|FB_IAB/Orca-Android).(?:FBAV)(?:[ /]([\d\.]+))?'
+- regex: '(?:MessengerForiOS|MESSENGER|FB_IAB/Orca-Android)\.(?:FBAV)(?:[ /]([\d\.]+))?'
   name: 'Facebook Messenger'
   version: '$1'
 
 # Facebook Messenger Lite
-- regex: '(?:mLite|MessengerLite(?:ForiOS)?).(?:FBAV)(?:[ /]([\d\.]+))?'
+- regex: '(?:mLite|MessengerLite(?:ForiOS)?)\.(?:FBAV)(?:[ /]([\d\.]+))?'
```

Comment:
```suggestion
- regex: '(?:mLite|MessengerLite(?:ForiOS)?).*FBAV(?:[ /]([\d\.]+))?'
```

Reply:
Also, you should move `Facebook Messenger Lite` before `Facebook Messenger`.

Reply:
Sure, I can move that if you want! I was trying to not make any unrelated changes in this PR.

Reply:
Or at least if this was `.` originally I'd rather go back to that than introduce `.*` unless there's a reason

Reply:
1de135493ef5767cd9433bc50afb4cfe1c2b6d2b

Reply:
And I moved FB Messenger Lite before Messenger: 0da6dca6

I think that's all. Thanks!